chromium/product.yml

```diff
@@ -5,3 +5,9 @@ type: product
 benchmark_root: "./guide"
 
 profiles_root: "./profiles"
+
+cpes:
+  chromium:
```

Comment:
nitpick: please make this `- chromium`.

Reply:
I can make this change, but I don't see the point in making it a list of CPEs.
Is this request aesthetic or functional?

Reply:
both. It makes it easier to read a file full of them.

Reply:
Made the change in c16efcd